pkg/controller/storageversionmigrator/storageversionmigrator.go

```diff
@@ -339,6 +336,47 @@ func (svmc *SVMController) sync(ctx context.Context, key string) error {
 	return nil
 }
 
+func isAllowableError(k8sError error) bool {
+	switch {
+	case apierrors.IsConflict(k8sError):
+		return true
+	case apierrors.IsNotFound(k8sError):
+		return true
+	default:
+		return false
+	}
+}
+
+func isRetriableError(k8sError error) bool {
+	switch {
+	case apierrors.IsServerTimeout(k8sError):
+		return true
+	case apierrors.IsTooManyRequests(k8sError):
+		return true
```

Comment:
We probably should use this to backoff for the seconds indicated via `SuggestsClientDelay`.